src/Elm/Parser/Tokens.elm

```diff
@@ -261,21 +260,11 @@ excludedOperators =
 
 allowedOperatorTokens : List Char
 allowedOperatorTokens =
-    [ '+', '-', ':', '/', '*', '>', '<', '=', '/', '&', '^', '%', '|', '!', '.', '#', '$', '≡', '~', '?', '@' ]
-
-
-allowedPrefixOperatorTokens : List Char
-allowedPrefixOperatorTokens =
-    ',' :: allowedOperatorTokens
+    [ '+', '-', ':', '/', '*', '>', '<', '=', '&', '^', '%', '|', '!', '.', '?' ]
```

Comment:
I left the `%` in there because the compile does it
https://github.com/elm/compiler/blob/2f6dd29258e880dbb7effd57a829a0470d8da48b/compiler/src/Parse/Symbol.hs#L77-L79
but I'm not sure if we should not remove it. Is it used in any operator I forgot about?

Reply:
Ah, it could be to give better error messages:
```elm
> 1 % 2
-- UNKNOWN OPERATOR ------------------------------------------------------- REPL

Elm does not use (%) as the remainder operator:

3|   1 % 2
       ^
If you want the behavior of (%) like in JavaScript, switch to:
<https://package.elm-lang.org/packages/elm/core/latest/Basics#remainderBy>

If you want modular arithmetic like in math, switch to:
<https://package.elm-lang.org/packages/elm/core/latest/Basics#modBy>

The difference is how things work when negative numbers are involved.
```

Reply:
Hmm, I'd say we remove it. Maybe we can re-introduce it later if we decide to give better error messages ourselves.

It looks like `(!)`, `(.)` still get marked as valid operators. Would removing their support go through updating this list?
(I won't mind having this part refined in a later PR, doesn't have to be done here).

Reply:
It's a bit more complicated than that. `(.)` and `(!)` are valid, but I think so are combinations like `(.!.)`, I think it's a remnant of the 1.18 where you could define operators from a list of valid characters.
In this codebase, it's used for parsing operators in expressions but it's also used for parsing the `infix` declaration that the core packages can still use.

But yes, I think we should fix those by replacing `operatorTokenFromList` by a list of known operators, maybe in a follow up PR, this one is getting crowded.